JS-Structure/js-structure_shapes/index.js

```diff
@@ -6,18 +8,21 @@ const circle = document.createElement("div");
 circle.classList.add("circle");
 circle.addEventListener("click", () => {
   circle.style.backgroundColor = "#ccc";
+  getRandomColor();
```

Comment:
Here I think the idea is to assign the value returned by the funtion to the circle.style.background, that means, instead of the #ccc it would receive the value returned by getRandomColor.

Reply:
ok, changes ☺️